packages/coze-js/src/ws-tools/wavtools/lib/wav_recorder.d.ts

```
@@ -0,0 +1,167 @@
+    sampleRate: number;
+    outputToSpeakers: boolean;
+    debug: boolean;
+    _deviceChangeCallback: () => Promise<void>;
+    _devices: any[];
+    stream: any;
+    processor: any;
+    source: any;
+    node: any;
+    recording: boolean;
+    _lastEventId: number;
+    eventReceipts: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)